test/molecule-role/molecule/swarm/files/docker-compose.yml

```diff
@@ -0,0 +1,89 @@
+---
+version: '3'
+
+services:
+
+  zookeeper:
+    image: wurstmeister/zookeeper
+    ports:
+      - 2181:2181
+    deploy:
+      placement:
+        constraints: [ node.role == manager ]
+      resources:
+        limits:
+          memory: 1G
+
+  kafka:
+    image: wurstmeister/kafka:2.12-2.3.1
+    ports:
+      - 9092:9092
+    depends_on:
+      - zookeeper
+    environment:
+      KAFKA_ADVERTISED_HOST_NAME: kafka
+      KAFKA_ADVERTISED_PORT: 9092
+      KAFKA_ZOOKEEPER_CONNECT: zookeeper:2181
+      KAFKA_CREATE_TOPICS: "sts_connection_beat_events:1:1,sts_generic_events:1:1,sts_state_events:1:1,sts_multi_metrics:1:1,sts_correlate_endpoints:1:1,sts_topo_process_agents:1:1,sts_trace_events:1:1,sts_topology_events:1:1"
+    healthcheck:
+      test: ["CMD-SHELL", "/opt/kafka/bin/kafka-topics.sh --list --zookeeper zookeeper | grep -q sts_trace_events"]
+      interval: 1s
+      timeout: 10s
+      retries: 10
+    deploy:
+      placement:
+        constraints: [ node.role == manager ]
+      resources:
+        limits:
+          memory: 1G
+
+  receiver:
+    image: "quay.io/stackstate/stackstate-receiver:${STACKSTATE_BRANCH}"
+    ports:
+      - 7077:7077
+      - 1618:1618
+    depends_on:
+      - kafka
+    environment:
+      KAFKA_BROKERS: kafka:9092
+    deploy:
+      placement:
+        constraints: [ node.role == manager ]
+      resources:
+        limits:
+          memory: 1G
+
+  topic-api:
+    image: "quay.io/stackstate/stackstate-topic-api:${STACKSTATE_BRANCH}"
+    ports:
+      - 7070:7070
+    depends_on:
+      - kafka
+    environment:
+      KAFKA_BROKERS: kafka:9092
+      ZOOKEEPER_QUORUM: kafka
+    deploy:
+      placement:
+        constraints: [ node.role == manager ]
+      resources:
+        limits:
+          memory: 1G
+
+  stackstate-agent:
+    image: docker.io/stackstate/stackstate-cluster-agent-test:${AGENT_VERSION}
+    deploy:
+      placement:
+        constraints: [ node.role == manager ]
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock:ro
+      - /etc/passwd:/etc/passwd:ro
+      - /sys/kernel/debug:/sys/kernel/debug
+    environment:
+      STS_API_KEY: "API_KEY"
+      STS_STS_URL: "http://receiver:7077/stsAgent"
+      STS_COLLECT_SWARM_TOPOLOGY: "true"
+      STS_LOG_LEVEL: "debug"
+      STS_LOG_TO_CONSOLE: "true"
+      STS_DOCKER_SWARM: "true"
```

Comment:
same here, should just be `DOCKER_SWARM`

Reply:
same as above.